addons/account/views/account_move_views.xml

```diff
@@ -544,7 +544,7 @@
                                 </div>
                                 <div class="row">
                                     <div class="col-6">
-                                        <span><field name="amount_total" widget='monetary'/></span>
+                                        <span><field name="amount_total_signed" widget='monetary' options="{'currency_field': 'currency_id'}"/></span>
```

Comment:
My bad, I went too fast while reading; you're totally right!

```suggestion
                                        <span><field name="amount_total_in_currency_signed" widget='monetary'/></span>
```

The field `amount_total_signed` is in foreign currency though (probably why you added the options?). The signed version of `amount_total` is `amount_total_in_currency_signed`